sync.py

```diff
@@ -120,27 +164,43 @@ def check_unsynced_builds(tagged_builds, packages_to_track):
                 print( ("%s needs to be updated to %s") % (build['package_name'], build['nvr']) )
                 unsynced_builds.append(build)
             # TODO: Ideally we should keep this directory and fetch latest tags to avoid repeated clones
-            os.system("rm -rf /tmp/" + build['package_name'])
+            os.system("rm -rf " + codir)
+    os.system("rm -rf " + corootdir)
     return unsynced_builds
 
 def check_unsynced_modules(tagged_builds, modules_to_track):
     """
     Look for modules that are not synced with centos streams.
     """
     unsynced_builds = []
-    for build in tagged_builds:
+    corootdir = "/tmp/centos-sync-mod-" + str(os.getpid()) + "/"
+    print("Using tmp dir:", corootdir)
+    os.makedirs(corootdir)
+    for build in sorted(tagged_builds, key=lambda x: x['package_name']):
         if build['package_name'] in modules_to_track:
-            os.system("rm -rf /tmp/" + build['package_name'])
-            # import ipdb; ipdb.set_trace();
-            repo = git.Repo.clone_from("https://git.centos.org/modules/" + build['package_name'] + ".git", "/tmp/" +
-            build['package_name'])
-            
+            codir = corootdir + build['package_name']
+            os.system("rm -rf " + codir)
```

Comment:
I earlier used `rm -rf` as a quick way to delete entire directories, but I believe there are better ways to do it in python.